src/seam-connect/client.ts

```diff
@@ -111,4 +126,51 @@ export class Seam extends Routes {
       throw error
     }
   }
+
+  static async getClientSessionToken(
+    ops: CSTParams
```

Comment:
Can we make this `CSTParams & SeamClientOptions`? This way users can just spread the same options whenever they call `new Seam` or `Seam.anyStaticMethod`?

Reply:
Actually, we might just need

```suggestion
    ops: SeamClientOptions & { userIdentifierKey?: string | undefined | null }
```

Reply:
That's ok to have it's own type the, but we should make sure to accept and handle the options that deal with the "global" client behavior. I think it is only missing `axiosOptions`.